app/utils/generalUtils.ts

```diff
@@ -48,6 +48,10 @@ export function validateEmail(email: unknown): email is string {
   return typeof email === "string" && email.length > 3 && email.includes("@")
 }
 
+export function stripHtml(text: string) {
+  return text.replace(/<\/?[^>]+(>|$)/g, "")
```

Comment:
In general, I would prefer to use an existing library for this kind of thing, e.g. https://www.npmjs.com/package/string-strip-html (haven't looked into it at all but I assume it does what it says it does). Handling HTML tags in strings isn't actually all that straightforward. Here is a classic post on the topic: https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags/1732454

Reply:
I made a ticket for this! Should be simple